HW0/P4/P4.py

```diff
@@ -1,7 +1,7 @@
 import numpy as np
 import matplotlib.pyplot as plt
 from mpl_toolkits.mplot3d import Axes3D
-
+import time
```

Comment:
do you need this line? i didn't see you use anything from this package in this file